client/src/components/PhotoContainerEntry.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PhotoModContainerStyle, PhotoModImageStyle } from './styledComponents.jsx';
+
+const PhotoContainerEntry = ({siteData, pictures, setModalOn, picIndex, setCurrentPicIndex}) => {
+
+  const pceClickHandler = () => {
```

Comment:
You might consider renaming pceClickHandler, but it is nice that you differentiate between other possible click handlers.

Reply:
Good point.  photoContainerEntryClickHandler is a keyboard-ful, but it is more descriptive.